xml/docu_styleguide.contributors.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE appendix
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<appendix xmlns="http://docbook.org/ns/docbook"
+          xmlns:xi="http://www.w3.org/2001/XInclude"
+          xmlns:xlink="http://www.w3.org/1999/xlink"
+          version="5.0" xml:id="sec-contributors">
+ <title>Contributors</title>
+
+ <para>
+  For a list of persons who contributed to this documentation, visit
```

Comment:
```suggestion
  For a list of the people who contributed to this documentation, visit
```

Reply:
I think this is a little less formal. I am only proposing it and am quite happy to be over-ruled. My 2nd preference would be "the persons", with a definite article.

Reply:
Hm. It's a Style Guide and we're a slightly stuck-up doc team. So, the current wording's fine, no?^^

Reply:
Nobody would ever say it. That's all, really.

Reply:
Nitpicking alert: I suggest `people` instead of `the people`. The definite `the people` implies we list *all* persons who ever contributed, which we can't because we lack some of the history anyway.

Reply:
```suggestion
  For a list of people who contributed to this documentation, visit
```